core/src/main/java/io/ultrabrew/metrics/data/TagSetsHelper.java

```diff
@@ -4,7 +4,10 @@
 
 package io.ultrabrew.metrics.data;
 
-class TagSetsHelper {
+import java.io.Serializable;
+import java.util.Comparator;
+
+class TagSetsHelper implements Comparator<Integer>, Serializable {
```

Comment:
Probably should rename the class to `TagSetComparator`.

Why does this implement `Serializable`?

Reply:
It is generally a good idea for comparators to also implement Serializable, as they may be used as ordering methods in serializable data structures (like TreeSet, TreeMap). In order for the data structure to serialize successfully, the comparator (if provided) must implement Serializable. I agree on the class name to be more specific.